posthog/hogql_queries/insights/trends/aggregation_operations.py

```diff
@@ -0,0 +1,147 @@
+from typing import List
+from posthog.hogql import ast
+from posthog.hogql.parser import parse_expr, parse_select
+from posthog.hogql_queries.utils.query_date_range import QueryDateRange
+from posthog.schema import ActionsNode, EventsNode
+
+
+class QueryAlternator:
```

Comment:
Might be good to leave a line of prose for the future traveller, as to what this class does, without having to go through all the code.